.github/actions/base-setup/action.yml

```diff
@@ -0,0 +1,43 @@
+name: Setup environment
+description: Setup base environment for Quetz frontend
+inputs:
+  frontend_folder:
+    description: Folder in which the frontend repository is checked out
+    required: true
+  server_folder:
+    description: Folder in which the server will be checked out
+    required: false
+    default: "server"
+
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: mamba-org/quetz
+        ref: master
+        path: ${{ inputs.server_folder }}
+
+    - uses: conda-incubator/setup-miniconda@v2
```

Comment:
Do you think we could use our micromamba action here?

Reply:
https://github.com/mamba-org/provision-with-micromamba

Reply:
Thanks for the suggestion.

Is it possible to set up package caching with that action?